docs/Friendly_User/Applications/vasp.md

```diff
@@ -1,50 +1,36 @@
-## VASP modules
+## VASP modules on Kestrel
 
-Section coming soon.
-
-## Running VASP
-
-We have found that it is optimal to run an intel toolchain build of VASP using cray-mpich-abi at runtime. Cray-mpich-abi has several dependencies on cray network modules, so the easiest way to load it is to first load ```PrgEnv-intel``` and then swap the default cray-mpich module for the cray-mpich-abi ```module swap cray-mpich cray-mpich-abi```. You must then load your intel compilers and math libraries, and unload cray's libsci. A sample script showing all of this is in the dropdown below.
+There are modules for CPU builds of VASP 5.4.4 and VASP 6.3.2 each with solvation, transision state tools, and BEEF-vdW functionals. These modules can be loaded with ```module load vasp/5.4.4``` or ```module load vasp/6.3.2```. A sample job script is shown below.
 
 !!! Note
     It is necessary to specify the launcher using srun --mpi=pmi2 
 
-??? Sample job script for your own vasp build
+??? example "Sample job script: using modules"
 
     ```
     #!/bin/bash
     #SBATCH --nodes=2
     #SBATCH --tasks-per-node=104
     #SBATCH --time=2:00:00
-    #SBATCH --mem=0 # ensures you are given all the memory on a node
+    #SBATCH --account=<your-account-name>
+    #SBATCH --job-name=<your-job-name>
 
-    # Load cray-mpich-abi and its dependencies within PrgEnv-intel, intel compilers, mkl, and unload cray's libsci
-    source /nopt/nrel/apps/env.sh
-    module purge
-    module load PrgEnv-intel
-    module swap cray-mpich cray-mpich-abi
-    module unload cray-libsci/22.10.1.2
-    module load intel-oneapi-compilers/2022.1.0  
-    module load intel-oneapi-mkl/2023.0.0-intel
+    source /nopt/nrel/apps/env.sh  #the need for this will eventually be removed
+    module load vasp/6.3.2
 
-    set -x 
-    export OMP_NUM_THREADS=1 #turns off multithreading
-    export VASP_PATH=/PATH/TO/YOUR/vasp_exe
-
-
-    srun --mpi=pmi2 ${VASP_PATH}/vasp_std |& tee out
-
-    #Note: it may be optimal to run with more processers per task, especially for heavier gw calculations e.g:
-    srun --mpi=pmi2 -ntasks 64 --ntasks-per-node=32 ${VASP_PATH}/vasp_std  |& tee out
+    srun --mpi=pmi2 vasp_std |& tee out
 
     ```
 
+## Compiling VASP yourself
+
+This section has reccomendations for toolchains to use for building and running VASP. Please read carefully before compilling on Kestrel's cray architecture.
 
-## Building VASP
+### Building VASP
 
 We recomend building vasp with a full intel toolchain and launching with the cray-mpich-abi at runtime. Additionally, you should build on a compute node so that you have the same architecture as at runtime:
```

Comment:
```suggestion
We recommend building vasp with a full intel toolchain and launching with the cray-mpich-abi at runtime. Additionally, you should build on a compute node so that you have the same architecture as at runtime:
```